client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts

```diff
@@ -98,7 +107,11 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
             .catch(err => {
                 contactLogger.error(`got the following error from the server: ${err}`, Severity.HIGH);
             })
-            .finally(() => setFormState(epidemiologyNumber, id, true))
+            .finally(() => {
+                ContactQuestioningSchema.isValid(formState).then(valid => {
```

Comment:
the name of the parameter formState is confusing because it seems to be the state for setFormState but its not
change to allInteractions or allContacts or contactsForm or contactsFormState or something like that

Reply:
changed to formData [here](575be28)